cannon/mipsevm/arch/arch64.go

```diff
@@ -69,6 +69,7 @@ const (
 	SysPrlimit64     = 5297
 	SysClose         = 5003
 	SysPread64       = 5016
+	SysStat          = 5004
```

Comment:
Do you mind adding a ticket to update the MTCannon spec to include this?

Reply:
Created a PR to update the spec - https://github.com/ethereum-optimism/specs/pull/445